src/layout-text.ts

```diff
@@ -1759,6 +1759,11 @@ export class Paragraph {
         const word = this.string.slice(wordStart, wordStart + wordLen);
         let wordGlyphs = wordCacheGet(font, word);
 
+        // Why the buffer is empty?
```

Comment:
@chearon On the first text shaping, the buffer is released, but I don't know where it was released, so regenerating a buffer, but it should not be a good solution?

Reply:
I'm not sure how the buffer could be empty here. It should only get destroyed after layout: 

https://github.com/chearon/dropflow/blob/75b1b5755efcc1f57e08f1d8e7cfcd1de74d912a/src/layout-flow.ts#L1497-L1504

Reply:
Could it be that harfbuzz clears the buffer in some cases?

Reply:
Let's fix this place like this？ and then find out the reason why the buffer was released in the future, and then modify it later.